### PR TITLE
Feature/AddChild

### PR DESCRIPTION
**Pull request by SandraCoburn**
# Description

Second draft of AddChild component. Added AddChildContainer to be able to have a protected route for this page. Changed some of the styling to get it closer to specs. 

## Type of change

Please delete options that are not relevant.

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Change Status

- [ ] Complete, tested, ready to review and merge
- [ ] Complete, but not tested (may need new tests)
- [X] Incomplete/work-in-progress, PR is for discussion/feedback

## Has This Been Tested

- [ ] Yes
- [X] No
- [ ] Not necessary

## Checklist

- [X] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] My code has been reviewed by at least one peer
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] There are no merge conflicts

### CODE REVIEW

**Review by JLC6290 — DISMISSED** [review]

**Review by Nick-Ohman — DISMISSED** [review]
Lets do this!

File: src/components/pages/AddChild/AddChild.less [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+}
+
+.ant-layout-sider {
+  background: #fff;
+  color: #fff;
+  line-height: 520px;
+  height: 100vh;
+  border-right: 2px solid gray;
+}
+// .ant-layout-header {
```
Comment (Schrese) [inline src/components/pages/AddChild/AddChild.less:61]: When we fix up everything, we can take out the comments as well

**Review by Nick-Ohman — DISMISSED** [review]

**Review by JLC6290 — DISMISSED** [review]

**Review by bramirez96 — DISMISSED** [review]
We got this!

File: src/components/pages/HomeScreen/HomeScreen.js [hunk 2]
```diff
@@ -1,62 +1,62 @@
 import React, { Component } from 'react';
 import { Link, BrowserRouter as Router, Route } from "react-router-dom"
 import { Layout, Menu, Button, Typography } from 'antd';
+import { Link } from 'react-router-dom';
```
Comment (Schrese) [inline src/components/pages/HomeScreen/HomeScreen.js]: Link is here twice

**Review by Nick-Ohman — APPROVED** [review]
great job getting this merge conflict resolved!

**Review by bramirez96 — APPROVED** [review]

**Review by Schrese — APPROVED** [review]

**Pull request merged**